treeherder/webapp/api/urls.py

```diff
@@ -115,6 +115,11 @@
     performance_data.PerformanceAlertSummaryViewSet,
     basename='performance-alert-summaries',
 )
+default_router.register(
```

Comment:
If I'm not mistaken this endpoint is no longer needed since you approached a different solution.